pallets/frag/src/weights.rs

```diff
@@ -0,0 +1,160 @@
+// This file is part of Substrate.
```

Comment:
How did you create the `weights.rs` file if the `benchmark.rs` file hasn't even been created?

Reply:
This is autogenerated

Reply:
I will ask you do actually work on that, so be good if you can get up to speed to the cause ;)